app/hooks/useClaimTo.tsx

```
@@ -0,0 +1,77 @@
+      if (!args[key]) {
+        throw new Error(`${key} is required`);
+      }
+    });
+
+    setIsProcessing(true);
+
+    try {
+      console.log({ args });
+
+      const transaction = claimTo({
+        contract: props.videoContract!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)